products/AOS-CX-Switching

```diff
@@ -0,0 +1,210 @@
+  regex: ^v(?<major>0|[1-9]\d*)_(?<minor>0|[1-9]\d*)_(?<patch>\d{1,3})_?(?<tiny>\d+)?$
+  # A liquid template using the captured variables from the regex above that renders the final version
+  # You can use liquid templating here
+  template: '..'
+
+
+# A list of releases, supported or not
+# Newer releases go on top of the list, in order
+releases:
+  - releaseCycle: "10.09.xxxx"
+    releaseLabel: ""
+    eol: 2022-12-22
+    support: 2022-12-22
+    release: 2021-12-09
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.08.xxxx"
+    releaseLabel: ""
+    eol: 2022-07-14
+    support: 2022-07-14
+    release: 2021-07-15
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.07.xxxx"
+    releaseLabel: ""
+    eol: 2022-07-15
+    support: 2022-07-15
+    release: 2021-07-15
+    latest: "10.09.10"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.06.xxxx"
+    releaseLabel: ""
+    eol: 2025-11-25
+    support: 2025-11-25
+    release: 2020-11-30
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.05.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2020-06-30
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.04.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2019-11-1
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: 
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.03.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2019-06-19
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+
+# Set an icon for the product from https://simpleicons.org/
+# If the icon is not available on simpleicons, set it to "NA"
+# As an example, https://simpleicons.org/?q=opensuse links to
+# https://simpleicons.org/icons/opensuse.svg and https://simpleicons.org/icons/opensuse.pdf
+# So the slug is `opensuse` (the SVG filename without extension).
+iconSlug: ministryofmagic
```

Comment:
i couldnt see any icon for ministryofmagic in https://simpleicons.org/ , if you dont have any icon in there just
make this line as
iconSlug: NA